Cargo.toml

```diff
@@ -1349,6 +1349,7 @@ substrate-test-runtime-client = { path = "substrate/test-utils/runtime/client" }
 substrate-test-runtime-transaction-pool = { path = "substrate/test-utils/runtime/transaction-pool" }
 substrate-test-utils = { path = "substrate/test-utils" }
 substrate-wasm-builder = { path = "substrate/utils/wasm-builder", default-features = false }
+subwasmlib = { git = "https://github.com/chevdor/subwasm", rev = "v0.21.3" }
```

Comment:
`chevdor/subwasm` doesn't release `0.21.3` to https://crates.io/crates/subwasmlib